### ISSUE TITLE
Connection error

### ISSUE THREAD

**Issue by seghier** [kyamagu/photoshop-connection#29]
Hello
Why this error?

`1. Error running script: Expected 43348 bytes, received 7012 bytes, password incorrect? [8:1]`

The password is correct and the hello window appears

**Comment by alexeykomp** [comment]
Indeed, there appears to be an error, when trying to receive a relatively large amount of data from Photoshop. E.g. with get_layer_thumbnail(max_width=200, max_height=200). The amount of data received is random and usually lower than expected.   
```
File "/venv3.12/lib/python3.12/site-packages/photoshop/protocol.py", line 169, in receive
    len(body) == length
AssertionError: Expected 128836 bytes, received 97988 bytes, password incorrect?
AssertionError: Expected 128836 bytes, received 81656 bytes, password incorrect?

```
From the 3rd-5th attempt, It usually manages to receive the whole data. 
I assume some packets are getting lost during transportation for whatever reason. 
I'm not sure how to debug this.
